src/test/java/co/rsk/bitcoinj/script/RedeemScriptValidatorTest.java

```diff
@@ -384,4 +384,102 @@ public void isOpnN_invalid_opcode() {
         ScriptChunk chunk = new ScriptChunk(ScriptOpCodes.OP_DROP, null);
         Assert.assertFalse(RedeemScriptValidator.isOpN(chunk));
     }
+
+    @Test
+    public void hasP2shErpRedeemScriptStructure_whenEmptyScript_shouldReturnFalse() {
+        Script emptyScript = new Script(new byte[]{});
+
+        boolean actualHasP2shErpRedeemScriptStructure = RedeemScriptValidator.hasP2shErpRedeemScriptStructure(
+            emptyScript.getChunks());
+
+        Assert.assertFalse(actualHasP2shErpRedeemScriptStructure);
```

Comment:
```suggestion
        List<ScriptChunk> emptyScriptChunks = emptyScript.getChunks();
        Assert.assertFalse(RedeemScriptValidator.hasP2shErpRedeemScriptStructure(emptyScriptChunks));
```

wdyt? looks easier to me

Reply:
actually we could import the class so this looks like

```suggestion
        List<ScriptChunk> emptyScriptChunks = emptyScript.getChunks();
        Assert.assertFalse(hasP2shErpRedeemScriptStructure(emptyScriptChunks));
```

Reply:
same in the other tests

Reply:
Updated